DEMS-Common/src/UDP/Message.java

```diff
@@ -43,26 +43,26 @@ of this software and associated documentation files (the "Software"), to deal
      */
     private OperationCode m_Code;
     private int m_SeqNum;
-    private Location m_Location;
+    private RegisteredReplica m_Location;
     private String m_Data;
     private InetAddress m_Addr;
     private int m_Port;
 
     // @warning max data.length is 2048 bytes !
     // Main entry point for initiating communication, See Socket for an example
     public Message(OperationCode code, int seq, String data, AddressBook addrInfo) throws Exception {
-        if(data.length() >= 2048){
+        if (data.length() >= 2048) {
             throw new Exception("data payload is too large!");
         }
-        
+
         this.m_Code = code;
         this.m_SeqNum = seq;
-        this.m_Location = Location.INVALID;
+        this.m_Location = RegisteredReplica.EVERYONE;
```

Comment:
okay it's embedded in message

Reply:
you can set the value when u need it but default, its unused